pyproject.toml

```diff
@@ -86,7 +86,9 @@ output-formatting = ['jinja2>=3.1.2', 'tabulate>=0.9.0']
 clipboard = ['PyQt5>=5.15.8', 'qtpy>=2.3.0']
 compression = ['zstandard>=0.19.0']
 consortium-standard = ['dataframe-api-compat>=0.1.7']
-all = ['beautifulsoup4>=4.11.2',
+all = ['adbc-driver-postgresql>=0.8.0',
```

Comment:
Would be good to add these to the `postgresql` and `sql-other` extras above as relevant